src/accountsdb/readme.md

```diff
@@ -174,10 +174,12 @@ to support disk-based account references, we created a general purpose
 disk allocator which creates memory from mmap-ing files stored on disk.
 
 ```zig
-// files are created using `data/test-data/tmp_{i}` format where `i` is 
-// incremented by one for each alloc call.
-var allocator = try DiskMemoryAllocator.init("data/test-data/tmp");
-defer allocator.deinit(null);
+// files are created using `data/test-data/bin_{i}` format where `i` is 
+// incremented by one for each new allocation.
+var dma_dir = try std.fs.cwd().makeOpenPath("data/test-data");
+defer dma_dir.close();
+var dma_state = try DiskMemoryAllocator.init(dma_dir, logger);
+const dma = dma_state.allocator();
```

Comment:
can you add a sentence or two on how the disk allocator works (ie, storing metdata at the end of the buffer)

Reply:
I'll get to updating this as soon as drew's review is resolved and the final details are confirmed.

Reply:
308ae941d92dc508b041b7af413cecd25b4b5ac1